src/batGc.mli

```diff
@@ -153,11 +159,17 @@ type control = Gc.control =
         first-fit policy, which can be slower in some cases but
         can be better for programs with fragmentation problems.
         Default: 0. @since 3.11.0 *)
-  }
+
+    window_size : int;
+    (** The size of the window used by the major GC for smoothing
+        out variations in its workload. This is an integer between
+        1 and 50.
+        Default: 1. @since 4.03.0 *)
+}
```

Comment:
We should probably export this field under a conditional test so that users can compile their code against the new Batteries but older OCaml version. Same thing for other `GC` primitives below. I can do the conditional stuff.